activerecord/lib/active_record/fixtures.rb

```diff
@@ -456,6 +456,31 @@ class FixtureClassNotFound < ActiveRecord::ActiveRecordError # :nodoc:
   # In the above example, 'base' will be ignored when creating fixtures.
   # This can be used for common attributes inheriting.
   #
+  # == Composite Primary Key Fixtures
+  #
+  # If a model has a composite primary key, both composite key columns should
```

Comment:
Nit: since theoretically a composite key could have more than 2 columns
```suggestion
  # If a model has a composite primary key, all composite key columns should
```